plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/ui/AppEngineStandardProjectConvertCommandHandler.java

```diff
@@ -38,26 +40,33 @@
 
   @Override
   public Object execute(ExecutionEvent event) throws ExecutionException {
+    Shell shell = HandlerUtil.getActiveShellChecked(event);
     try {
       IProject project = ProjectFromSelectionHelper.getProject(event);
       if (project == null) {
         throw new NullPointerException("Convert menu enabled for non-project resources");
       }
 
+      // check if Cloud SDK is available, prompts to configure if not
+      if (CloudSdkPrompter.getCloudSdkLocation(shell) == null) {
```

Comment:
I prefer moving this down below after `if (hasAppEngineFacet())`, because the first two checks are sort of assertions that should always be true.